test/files/neg/t771.check

```diff
@@ -1,4 +1,4 @@
-t771.scala:4: error: trait Iterator is abstract; cannot be instantiated
+t771.scala:4: error: trait Iterator is trait; cannot be instantiated
```

Comment:
Here, the word "abstract" is explanatory. It already says it is a trait, so "trait" is redundant.